pkgs/applications/misc/inkcut/default.nix

```diff
@@ -43,7 +62,19 @@ buildPythonApplication rec {
   ];
 
   dontWrapQtApps = true;
-  makeWrapperArgs = [ "\${qtWrapperArgs[@]}" ];
+  makeWrapperArgs = [
+    "\${qtWrapperArgs[@]}"
+    "--unset PYTHONPATH"
```

Comment:
Why? People might set `PYTHONPATH` on purpose, and I imagine this could lead to a very long and frustrating debugging session.

Reply:
Thanks for the review!

Inkcut starts fine from the commandline, but didn't start when invoked from the inkcut extension. This fixed that, so apparently there's something on the PYTHONPATH passed along via Inkscape that conflicts with what inkcut expects.

I'll try to dig a bit deeper to document exactly what causes the conflict, and if I don't see a way to resolve the conflict perhaps see if we can clear the PYTHONPATH in the extension instead of in inkcut's main wrapper.

Reply:
It just silently fails to start when I don't clear the PYTHONPATH when calling from Inkscape...

I've reduced the scope of the hack by moving it from the inkcut wrapper script to the extension. This also makes it more visible, as it's in a clearly-named patch. Still not an entirely satisfying 'solution', but I think it should be sufficient for now.

Reply:
Did you try to print `PYTHONPATH` in the wrapper? It would be interesting to know what it is actually set to / where the value is coming from.

Reply:
Figured it out!

The PYTHONPATH contained the Inkscape extension directory, which contained an 'inkcut.py' with some extension utilities. When running inkcut from the extension, it would find that `inkcut` instead of the application itself...

Proposed renaming the utilities file upstream in https://github.com/inkcut/inkcut/pull/292, included that patch here for now.

Reply:
Great :) I'm wondering why our `PYTHONPATH` includes the inkscape extension directory. Apparently this is not normal, since otherwise upstream would have discovered the issue earlier. But that's getting out of scope for this PR.

Thank you for investigating!